Assets/Scripts/Singletons/CurrentGameObjects.cs

```diff
@@ -81,6 +88,38 @@ public void addObjectsCollected(GameObject aPlant)
         return objectsCollected;
     }
 
+    // Return just potions created by the user
+    public List<Potion> GetPotionsCreated()
+    {
+        List<Potion> potions = new List<Potion>();
+
+        // Create Strength Potion
+        List<CollectablePlantsEnum> strengthPotionPlantIngredients = new List<CollectablePlantsEnum>
+        { CollectablePlantsEnum.Rose, CollectablePlantsEnum.Wheatgrass, CollectablePlantsEnum.ConeFlower};
+
+        Potion strengthPotion = new Potion("Strength Potion", 1000, strengthPotionPlantIngredients);
+
+        // Create Speed Potion
+        List<CollectablePlantsEnum> speedPotionPlantIngredients = new List<CollectablePlantsEnum>
+        { CollectablePlantsEnum.Ginger, CollectablePlantsEnum.ConeFlower, CollectablePlantsEnum.Spearmint};
+
+        Potion speedPotion = new Potion("Speed Potion", 3000, speedPotionPlantIngredients);
+
+        // Create Sleep Potion
+        List<CollectablePlantsEnum> sleepPotionPlantIngredients = new List<CollectablePlantsEnum>
+        { CollectablePlantsEnum.Lavender, CollectablePlantsEnum.Chamomile, CollectablePlantsEnum.Ginger};
+
+        Potion sleepPotion = new Potion("Sleep Potion", 2000, sleepPotionPlantIngredients);
+
+        potions.Add(strengthPotion);
+        potions.Add(speedPotion);
+        potions.Add(sleepPotion);
+
+        return potions;
+
+        // return this.potionsCreated;
```

Comment:
Remove comment.

Reply:
This is be the return once the potions are added into the list. all the other code in the method will be removed.